behat.yml.dist

```diff
@@ -131,6 +131,22 @@ default:
             filters:
                 tags: "@inventory && ~@javascript"
 
+        importexport:
+            contexts:
+                - Sylius\Bundle\ImportExportBundle\Behat\ImportExportContext
+                - Behat\MinkExtension\Context\MinkContext
+                - Sylius\Bundle\CoreBundle\Behat\HookContext
+                - Sylius\Bundle\AddressingBundle\Behat\AddressingContext
+                - Sylius\Bundle\CoreBundle\Behat\CoreContext
+                - Sylius\Bundle\PaymentBundle\Behat\PaymentContext
+                - Sylius\Bundle\ProductBundle\Behat\ProductContext
+                - Sylius\Bundle\TaxationBundle\Behat\TaxationContext
+                - Sylius\Bundle\TaxonomyBundle\Behat\TaxonomyContext
+                - Sylius\Bundle\WebBundle\Behat\WebContext
+                - Sylius\Bundle\MoneyBundle\Behat\MoneyContext
+            filters:
+                tags: "@importexport && ~@javascript"
```

Comment:
Don't do anything with javascript in your suite as javascript tags are included in javascript profile only.